### DISCUSSION TITLE
Place and End Cloud PC Review

### DISCUSSION THREAD

**Discussion by Shane6712** [Windows365Management/PSCloudPC#29] [Ideas]
Hello, thank you so much for the work on PSCloudPC - are there any plans to add place /end cloud PC under review commands? Or even create additional shareable snapshots?

**Comment by Ruthhl3ss** [comment]
Hello!

Your first question about the place/end cloud PC under review does not feature this command, right?

https://github.com/Windows365Management/PSCloudPC/blob/develop/PSCloudPC/Public/Invoke-CPCEndGracePeriod.ps1

You want to place a Cloud PC under review and not end it. 

Secondly, you would like to create snapshots by using a PS command?

Thanks,
Niels
